### PR TITLE
Feature/hnj 10 creating header markup

### PR DESCRIPTION
**Pull request by zarinaazamatova**

### CODE REVIEW

**Comment by Alex-2kZharkov** [comment]
@zarinaazamatova Please don't forget to change commit descriptions

image

File: src/components/Header/header.styles.ts [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+export const StyledDivTop = styled.div`
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  align-items: center;
+  margin: 0px 80px 0px 80px;
+  box-sizing: border-box;
+  width: ;
+`;
+export const StyledDivLacation = styled.div`
```
Comment (Alex-2kZharkov) [inline src/components/Header/header.styles.ts:28]: `StyledLocation`

File: src/components/Header/header.styles.ts [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    box-sizing: border-box;
+    background-color: ${theme.bgColors.secondary};
+    font-family: 'Roboto', sans-serif;
+    font-size: 14px;
+  `};
+`;
+
+/* Top
+   ========================================================================== */
+export const StyledDivTop = styled.div`
```
Comment (Alex-2kZharkov) [inline src/components/Header/header.styles.ts:19]: Please, remove `DivPrefix`, it does not add any value

File: src/components/Header/header.styles.ts [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: 0px 0px 2px 0px;
+`;
+export const StyledDivCity = styled.div`
+  ${({ theme }) => css`
+    color: ${theme.textColors.orange};
+    font-size: ${theme.fontSizes.sm};
+    font-weight: 300;
```
Comment (Alex-2kZharkov) [inline src/components/Header/header.styles.ts:39]: `300` should come from the `theme` object

File: src/components/Header/header.styles.ts [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+  position: relative;
+  display: flex;
+  margin: 0px 0px 0px 310px;
+`;
+export const StyledDivMenu = styled.div`
+  ${({ theme }) => css`
+    margin: 0px 0px 0px 20px;
+    color: ${theme.textColors.darkGrey};
+    font-size: ${theme.fontSizes.sm};
+    font-weight: 600;
```
Comment (Alex-2kZharkov) [inline src/components/Header/header.styles.ts:52]: `600` should come from the theme object

File: src/components/Header/header.styles.ts [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+export const StyledSpanCounter = styled.span`
+  display: block;
+  position: absolute;
+  z-index: 2;
+  bottom: 18px;
+  left: 15px;
+  min-width: 9px;
+  height: 6px;
+  padding: 2px;
+  font-size: 7px;
```
Comment (Alex-2kZharkov) [inline src/components/Header/header.styles.ts:126]: Same thing: use `theme` object for the size of Fonts

File: src/components/Header/header.styles.ts [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+  border-radius: 14px;
+  color: #fff;
+  background-color: #0a8449;
+`;
+/* Nav
+   ========================================================================== */
+
+export const StyledNav = styled.nav`
+  display: block;
+  margin: 0px 80px 0px 80px;
```
Comment (Alex-2kZharkov) [inline src/components/Header/header.styles.ts:138]: use shorter version `margin: 0px 80px;`

File: src/components/Header/header.styles.ts [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+`;
+export const StyledUl = styled.ul`
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+		padding: 0;
+		margin 10px 0 10px 0;
```
Comment (Alex-2kZharkov) [inline src/components/Header/header.styles.ts:145]: use shorter version `margin: 10px 0px;`

File: src/components/Header/header.styles.ts [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+  background-color: #0a8449;
+`;
+/* Nav
+   ========================================================================== */
+
+export const StyledNav = styled.nav`
+  display: block;
+  margin: 0px 80px 0px 80px;
+`;
+export const StyledUl = styled.ul`
```
Comment (Alex-2kZharkov) [inline src/components/Header/header.styles.ts:140]: `StyledList`

File: src/components/Header/header.styles.ts [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+		padding: 0;
+		margin 10px 0 10px 0;
+		width: auto;
+`;
+export const StyledLi = styled.li`
```
Comment (Alex-2kZharkov) [inline src/components/Header/header.styles.ts:148]: `StyledListItem`

File: src/components/Header/header.styles.ts [hunk 10]
```diff
@@ -0,0 +1,171 @@
...
+  display: flex;
+  align-items: center;
+  list-style: none;
+  font-size: 13px;
+  line-height: 1.14;
+  font-weight: 600;
```
Comment (Alex-2kZharkov) [inline src/components/Header/header.styles.ts:154]: `theme` object

File: src/components/Header/header.tsx [hunk 11]
```diff
@@ -0,0 +1,126 @@
...
+        <S.StyledDivUserInfo>
+          <Image src={user} alt="user" className="icon-user" width={16} height={16} />
+          <span>Войти</span>
+          <Image
+            src={arrow}
+            alt="arrow"
+            className="icon-arrow-down"
+            width={12}
+            height={15}
+            style={{ padding: '5px' }}
```
Comment (Alex-2kZharkov) [inline src/components/Header/header.tsx:59]: All of these inline styles should be used, ok, you will fix in the next task

**Review by Alex-2kZharkov — CHANGES_REQUESTED** [review]

File: .vscode/settings.json
[the remark this answers is not in the document]
Comment (zarinaazamatova) [reply .vscode/settings.json:1]: ✅

File: src/assets/png/logo.png
[the remark this answers is not in the document]
Comment (zarinaazamatova) [reply src/assets/png/logo.png:1]: ✅

File: src/assets/png/cart.png
[the remark this answers is not in the document]
Comment (zarinaazamatova) [reply src/assets/png/cart.png:1]: ✅

File: src/assets/svg/geolacation.tsx [hunk 12]
```diff
@@ -0,0 +1,30 @@
+const Geolocation = ({ width, height }: { width: number; height: number }) => {
```
[the remark this answers is not in the document]
Comment (zarinaazamatova) [reply src/assets/svg/geolacation.tsx:1]: ✅

File: src/assets/svg/geolacation.tsx [hunk 13]
```diff
@@ -0,0 +1,30 @@
...
+      </g>
+      <defs>
+        <clipPath id="clip0_105_1114">
+          <rect width="140" height="140" fill="white" transform="translate(0 0.5)" />
+        </clipPath>
+      </defs>
+    </svg>
+  );
+};
+export default Geolocation;
```
[the remark this answers is not in the document]
Comment (zarinaazamatova) [reply src/assets/svg/geolacation.tsx:30]: ✅

File: src/components/Header/header.tsx [hunk 14]
```diff
@@ -0,0 +1,126 @@
...
+import user from '../../assets/png/user.png';
+import arrow from '../../assets/png/chevron-down.png';
+import heart from '../../assets/png/heart.png';
+import cart from '../../assets/png-files/cart.png';
+
+function Header() {
+  return (
+    <S.StyledDivContainer>
+      <S.StyledDivTop>
+        <S.StyledDivLacation>
```
[the remark this answers is not in the document]
Comment (zarinaazamatova) [reply src/components/Header/header.tsx:16]: ✅

File: src/components/Header/header.tsx [hunk 15]
```diff
@@ -0,0 +1,126 @@
...
+          <Image
+            src={arrow}
+            alt="arrow"
+            className="icon-arrow-down"
+            width={12}
+            height={15}
+            style={{ padding: '5px' }}
+          />
+        </S.StyledDivLacation>
+        <S.StyledDivBonusSide>
```
[the remark this answers is not in the document]
Comment (zarinaazamatova) [reply src/components/Header/header.tsx:28]: ✅

File: src/components/Header/header.tsx [hunk 16]
```diff
@@ -0,0 +1,126 @@
...
+          <S.StyledDivMenu>Бесплатная доставка от 500 ₽</S.StyledDivMenu>
+          <S.StyledDivMenu>Бонусная программа</S.StyledDivMenu>
+          <S.StyledDivMenu>Мобильное приложение</S.StyledDivMenu>
+        </S.StyledDivBonusSide>
+      </S.StyledDivTop>
+
+      <S.StyledHeaderCenter>
+        <Image src={logo} alt="arrow" className="icon-map" width={224} />
+        <S.StyledDivCatalog>
```
[the remark this answers is not in the document]
Comment (zarinaazamatova) [reply src/components/Header/header.tsx:37]: ✅

File: src/components/Header/header.tsx [hunk 17]
```diff
@@ -0,0 +1,126 @@
...
+            className="icon-list"
+            width={16}
+            height={16}
+            style={{ padding: '0 15px 0 0' }}
+          />
+          <S.StyledSpanCatalogTitle>Каталог</S.StyledSpanCatalogTitle>
+
+          {/* here will be a Form(input) from Asel */}
+        </S.StyledDivCatalog>
+        <S.StyledDivUserInfo>
```
[the remark this answers is not in the document]
Comment (zarinaazamatova) [reply src/components/Header/header.tsx:50]: ✅

File: src/components/Header/header.tsx (hunk 11 above)
Comment (zarinaazamatova) [reply src/components/Header/header.tsx:59]: ✅

**Review by Alex-2kZharkov — APPROVED** [review]
Excellent job! It was hard but you made it. Keep it up🙌

**Pull request merged**